test/__snapshot__/linter/no-double-equals-auto-fix.md

```diff
@@ -0,0 +1,88 @@
+# Kataw parser test case
+
+## Input
+
+`````js
+foo == bar;
+`````
+
+## Options
+
+### Parser Options
+
+`````js
+{ allowTypes: true, lint: { noDoubleEquals: true }, autofix: true }
+`````
+
+## Output
+
+### CST
+
+```javascript
+{
+    "kind": 122,
+    "directives": [],
+    "statements": [
+        {
+            "kind": 120,
+            "expression": {
+                "kind": 198,
+                "left": {
+                    "kind": 134299649,
+                    "text": "foo",
+                    "rawText": "foo",
+                    "flags": 96,
+                    "transformFlags": 0,
+                    "start": 0,
+                    "end": 3
+                },
+                "operatorToken": {
+                    "kind": 34620,
+                    "flags": 96,
+                    "transformFlags": 16,
+                    "start": 3,
+                    "end": 6
+                },
+                "right": {
+                    "kind": 134299649,
+                    "text": "bar",
+                    "rawText": "bar",
+                    "flags": 96,
+                    "transformFlags": 0,
+                    "start": 6,
+                    "end": 10
+                },
+                "flags": 96,
+                "transformFlags": 5120,
+                "start": 0,
+                "end": 10
+            },
+            "flags": 16,
+            "transformFlags": 4096,
+            "start": 0,
+            "end": 11
+        }
+    ],
+    "isModule": false,
+    "source": "foo == bar;",
+    "fileName": "__root__",
+    "flags": 0,
+    "transformFlags": 0,
+    "start": 0,
+    "end": 11
+}
+```
+
+### Printed
+
+```javascript
+
+```
+
+### Diagnostics
+
+```javascript
+✖ '==' may be unsafe if you are relying on type coercion - start: 3, end: 6
```

Comment:
@aladdin-add This rule has been auto-fixed but because of the test runner has not been fixed everything is seen as an error and it will not be printed. See #159